release/gcb/gcb_lib.sh

```diff
@@ -95,8 +95,16 @@ function make_istio() {
   sha256sum "${ISTIO_OUT}/istio-sidecar.deb" > "${OUTPUT_PATH}/deb/istio-sidecar.deb.sha256"
   cp        "${ISTIO_OUT}/istio-sidecar.deb"   "${OUTPUT_PATH}/deb/"
   cp        "${ISTIO_OUT}"/archive/istio-*z*   "${OUTPUT_PATH}/"
-  cp        "${ISTIO_OUT}"/archive/istioctl*.tar.gz "${OUTPUT_PATH}/"
-  cp        "${ISTIO_OUT}"/archive/istioctl*.zip "${OUTPUT_PATH}/"
+  ISTIOCTL_TARS=$(ls "${ISTIO_OUT}"/archive/istioctl*.tar.gz)
+  
+  echo $ISTIOCTL_TARS
+  
+  for i in $ISTIOCTL_TARS; do
+    sha256sum "${ISTIO_OUT}"/$i > "${OUTPUT_PATH}/$i.sha256"
```

Comment:
isn't this only the tar.gz, not the zip?

Reply:
@howardjohn yes, you are correct. I wanted to test the sha256sum operation inside the gate with the tar.gzs before modifying this PR to include the .zips.